templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.Mongo/backend/requirements_postaction.txt

```diff
@@ -1,5 +1,5 @@
 //{[{
-pymongo==3.10.1
-python-dotenv==0.13.0
+pymongo==3.11.1
```

Comment:
Think there is a typo in the pymongo version, this should be 3.11.0 according to https://pypi.org/project/pymongo/